### PR TITLE
Blog for Inspection Checks added for Deployment

### PR DESCRIPTION
**Pull request by subodhkumarWM**

### CODE REVIEW

File: website/blog/2020-04-22-deployments-made-better.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: "Deployment Experience made better!"
+author: Subodh Kumar
+---
+
+Deployment is a process which enables the developers to serve the apps to users.
```
Comment (prasanth-reddy-wavemaker-com) [inline website/blog/2020-04-22-deployments-made-better.md:6]: WaveMaker's 1-click deploy feature allows developers deploy their application to AWS, Azure, Google Cloud or Kubernetes cluster

File: website/blog/2020-04-22-deployments-made-better.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+title: "Deployment Experience made better!"
+author: Subodh Kumar
+---
+
+Deployment is a process which enables the developers to serve the apps to users. 
+With the latest release, WaveMaker has made the discovery of deployment failures in the application fast & easy.
```
Comment (prasanth-reddy-wavemaker-com) [inline website/blog/2020-04-22-deployments-made-better.md:7]: When user invokes Deploy operation WaveMaker trigger the maven command to build the project and generate artefacts for deployment. Build process includes angular build process to generated optimized javascript assets. During the build process there can be failures.

File: website/blog/2020-04-22-deployments-made-better.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+### Clear description of Errors
+* Each error includes details such as filename & exact line number which can cause deployment failure.
+* It also includes the [documentation](https://www.wavemaker.com/learn/app-development/dev-integration/inspection-framework#__docusaurus) link, to help the developer understand the issue better & help in looking for solutions.
+
+[![screenshot](/learn/assets/wm-blog-deploy-04-detail.png)](/learn/assets/wm-blog-deploy-04-detail.png)
+
+### Convenience
+
+* The validations are added as a part of Inspection framework, the user need not trigger Deployment to find out the issues in the project.
```
Comment (prasanth-reddy-wavemaker-com) [inline website/blog/2020-04-22-deployments-made-better.md:58]: Talk little bit about the inspection framework is built on existing opensource tech

File: website/blog/2020-04-22-deployments-made-better.md (hunk 1 above)
Comment (subodhkumarWM) [reply website/blog/2020-04-22-deployments-made-better.md:6]: Added

File: website/blog/2020-04-22-deployments-made-better.md (hunk 2 above)
Comment (subodhkumarWM) [reply website/blog/2020-04-22-deployments-made-better.md:7]: Added.

File: website/blog/2020-04-22-deployments-made-better.md (hunk 3 above)
Comment (subodhkumarWM) [reply website/blog/2020-04-22-deployments-made-better.md:65]: Added some points in the last section

**Pull request merged**